c3/optimizers/c3.py

```diff
@@ -244,14 +244,11 @@ def _log_one_dataset(
     ) -> None:
         seqs_pp = self.seqs_per_point
         m_vals = data_set["results"][:seqs_pp]
-        m_stds = np.array(data_set["result_stds"][:seqs_pp])
+        m_stds = np.array(data_set["results_std"][:seqs_pp])
         m_shots = data_set["shots"][:seqs_pp]
         sequences = data_set["seqs"][:seqs_pp]
         with open(self.logdir + self.logname, "a") as logfile:
-            logfile.write(
-                f"\n  Parameterset {ipar + 1}, #{count} of {len(indeces)}:\n"
-                f"{str(self.exp.pmap)}\n"
```

Comment:
Removing `str(self.exp.pmap)` is correct, but do we instead want to print the current parameters in human-readable form here? ` exp.str_parameters()` should do the trick.